src/main/java/seedu/address/commons/core/GuiSettings.java

```diff
@@ -24,15 +25,17 @@ public GuiSettings() {
         windowWidth = DEFAULT_WIDTH;
         windowHeight = DEFAULT_HEIGHT;
         windowCoordinates = null; // null represent no coordinates
+        attributeOrder = "ADDRESS>TAGS>PHONE>EMAIL";
     }
 
     /**
      * Constructs a {@code GuiSettings} with the specified height, width and position.
      */
-    public GuiSettings(double windowWidth, double windowHeight, int xPosition, int yPosition) {
+    public GuiSettings(double windowWidth, double windowHeight, int xPosition, int yPosition, String attributeOrder) {
         this.windowWidth = windowWidth;
         this.windowHeight = windowHeight;
         windowCoordinates = new Point(xPosition, yPosition);
+        this.attributeOrder = attributeOrder;
     }
 
     public double getWindowWidth() {
```

Comment:
Just a suggestion, can add documentation since these are public methods😉.

Reply:
Ok I have added on to the documentation for this method.